hash/fmtowns_cd.xml

```diff
@@ -1018,6 +1018,143 @@ User/save disks that can be created from the game itself are not included.
 		</part>
 	</software>
 	
+	<software name="eimmy">
+		<description>Eimmy to Yobanaide</description>
+		<year>1995</year>
+		<publisher>C's Ware</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="eimmy to yobanaide" sha1="4c321b593322b94cf84a68ad1021d0cf3b24b1c4" />
+			</diskarea>
+		</part>
+	</software>
+	
+	<software name="elfish">
+		<description>Elfish</description>
+		<year>1994</year>
+		<publisher>Fujitsu</publisher>
+		<info name="usage" value="Requires HDD installation"/>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="elfish" sha1="c0dd2d0c0caaea9098de8a550aac7b5005030f8f" />
+			</diskarea>
+		</part>
+	</software>
+	
+	<software name="elfishlt">
+		<description>Elfish Lite</description>
+		<year>1993</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="elfish" sha1="1667eb91450e756ae37910aa9291957430fc588b" />
```

Comment:
I'd guess the second one was meant to be called elfishlt.chd ;)

in the meanwhile, however, you can put them in elfish/ and elfishlt/
subfolders, respectively, and MAME won't see any conflict anymore...

   f.



On Wed, May 11, 2016 at 11:38 AM, Thomas Klausner <notifications@github.com>
wrote:

> In hash/fmtowns_cd.xml
> <https://github.com/mamedev/mame/pull/864#discussion_r62818439>:
>
> > +		<publisher>Fujitsu</publisher>
> > +		<info name="usage" value="Requires HDD installation"/>
> > +		<part name="cdrom" interface="fmt_cdrom">
> > +			<diskarea name="cdrom">
> > +				<disk name="elfish" sha1="c0dd2d0c0caaea9098de8a550aac7b5005030f8f" />
> > +			</diskarea>
> > +		</part>
> > +	</software>
> > +	
> > +	<software name="elfishlt">
> > +		<description>Elfish Lite</description>
> > +		<year>1993</year>
> > +		<publisher>Fujitsu</publisher>
> > +		<part name="cdrom" interface="fmt_cdrom">
> > +			<diskarea name="cdrom">
> > +				<disk name="elfish" sha1="1667eb91450e756ae37910aa9291957430fc588b" />
>
> Both elfish and elfishlt want to have a disk called "elfish.chd" in the
> file system, but with different checksums. That won't work, please rename
> one of them.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/mamedev/mame/pull/864/files/bf12928320cfc347050a76fb18e171570954270c#r62818439>
>